docs/native-modules-setup.md

```diff
@@ -14,17 +16,15 @@ This guide will help set you up with the Visual Studio infrastructure to author
 
 Make sure you have installed all of the [development dependencies](rnw-dependencies.md).
 
-If you're planning on writing in C++, you *must* install the [C++/WinRT Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=CppWinRTTeam.cppwinrt101804264).
```

Comment:
shouldn't we keep this on? or at least make it a recommendation?

Reply:
actually it might make sense for rnw-dependencices to install it

Reply:
AFAIK this only adds the C++/WinRT project templates to VS. Since the CLI is creating the project for them, they don't really need it.